### PR TITLE
Add Hydra mode and filename support to bench CSV output

### PR DESCRIPTION
**Pull request by teknico**
This does two things to the lwAftr `bench` command:
- it adds support for a different CSV data format, needed for running Hydra automated benchmarks;
- it makes the command emit data to a file instead of stdout (with support for choosing it), which is not reliable because of #463.

### CODE REVIEW

File: src/lib/lua/alt_getopt.lua [hunk 1]
```diff
@@ -24,8 +24,6 @@ local type, pairs, ipairs, io, os = type, pairs, ipairs, io, os
 module (...)
 
 local function convert_short2long (opts)
-   local i = 1
-   local len = #opts
```
Comment (teknico) [inline src/lib/lua/alt_getopt.lua:28]: This is a completely unrelated cleanup, sorry.

File: src/program/lwaftr/bench/README [hunk 2]
```diff
@@ -2,9 +2,28 @@ Usage: bench CONF IPV4-IN.PCAP IPV6-IN.PCAP
...
   -h, --help
                              Print usage information.
+  -y HYDRA, --hydra
+                             Hydra mode: emit CSV data in the format expected
+                             by the Hydra reports. For instance:
+
+                               benchmark,snabb,id,score,unit
+
+                             rather than the default:
+
```
Comment (kbara) [inline src/program/lwaftr/bench/README:12]: Can we just kill off the old kind entirely? @wingo

**Comment by kbara** [comment]
LGTM once tests pass

File: src/program/lwaftr/csv_stats.lua [hunk 3]
```diff
@@ -65,17 +88,30 @@ function CSVStatsTimer:tick()
    local elapsed = engine.now() - self.start
    local dt = elapsed - self.prev_elapsed
    self.prev_elapsed = elapsed
-   self.file:write(('%f'):format(elapsed))
+   if not self.hydra_mode then
+      self.file:write(('%f'):format(elapsed))
+   end
```
Comment (teknico) [inline src/program/lwaftr/csv_stats.lua:93]: Admittedly the overall approach of conditionally changing the output depending on a flag, instead of doing it via object orientation, is not great: however it would have taken quite a bit of abstraction and decoupling for just a few small changes to the code.

**Review by teknico — COMMENTED** [review]
I preventively added a few comments to the changes for your own benefit. :-)

File: src/program/lwaftr/csv_stats.lua (hunk 3 above)
Comment (kbara) [reply src/program/lwaftr/csv_stats.lua:93]: I'm quite happy to have a few flags for doing that kind of thing, particularly in tooling outside the core of the system. :-)

**Comment by kbara** [comment]
LGTM, good catch on the docs.
